scripts/rtma3d/exrtma3d_gsianl.ksh

```diff
@@ -47,7 +47,7 @@ time_str2=`${DATE} "+%Y-%m-%d_%H_00_00" -d "${START_TIME}"`
 #----- enter working directory -------
 cd ${DATA}
 ${ECHO} "enter working directory:${DATA}"
-nc_diag_cat=/lfs/h2/emc/da/noscrub/edward.colon/save/RTMA/bin/ncdiag_cat_serial.x
+nc_diag_cat=${EXECrtma3d}/ncdiag_cat_serial.x
```

Comment:
@EdwardColon-NOAA 
Hi Ed,

Similar as my questions and comments above in "scripts/akrtma3d/exakrtma3d_gsianl.ksh". We need to build the exe ncdiag_cat_serial.x when building  the 3DRTMA pacakge, and we also need to include the source code of ncdiag in the pacakge.   Or just as Annette mentioned, loading mcdiag module might be alternative.

Thanks,
-Gang

Reply:
@GangZhao-NOAA I plan to load the ncdiag module.  I added it to the gsi script for my NCO-compliancy work, but did not have a chance to test it yet.  I should point out that RRFS is loading the ncdiag module instead of including the ncdiag code.